pricing-page-exercise/ece/index.html

```diff
@@ -1,12 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>HTML 5 Boilerplate</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/5e08b15075.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
+    <header>
+      <div class="navbar-container">
+          <div class="navbar-right">
+            <ul class="options">
+              <li class="options-right"><a class="navbar-links" href="#">Features</a></li>
+              <li class="options-right"><a class="navbar-links" href="#">Enterprise</a></li>
+              <li class="options-right"><a class="navbar-links" href="#">Support</a></li>
+              <li class="options-right"><a class="navbar-links" href="#">Pricing</a></li>
+            </div>
+            <div class="signup options-right"><a class="signup options-right" href="https://getbootstrap.com/docs/4.0/examples/pricing/#">Sign up</a></div>
+          <div class="navbar-left">
```

Comment:
It would make more sense to place this div before the .navbar-right div

Reply:
do you mean navbar-left or signup button?